tests/fast-integration/utils/index.js

```diff
@@ -1664,6 +1665,7 @@ module.exports = {
   k8sApply,
   k8sDelete,
   waitForK8sObject,
+  waitForK8sObjectWithWatch,
```

Comment:
You need to remove this line
```suggestion
  waitForK8sObjectWithWatch,
```
```suggestion
```

Reply:
Removed just before you wrote the comment. Sorry for not using drafts!